exp-player/addon/components/select-input/component.js

```diff
@@ -5,5 +5,6 @@ import layout from './template';
 export default Ember.Component.extend({
   layout,
   options: null,
-  value: null
+  value: null,
+  translate: true
```

Comment:
For future development, I am trying to encourage that all components document the kinds of parameters they expect to be passed in, as well as what those parameters are for.

We're using YUIDoc syntax for this. Here is an example: https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/components/osf-navbar/component.js#L34

One caveat: in the example above, you'll also notice that it includes some boilerplate ndeclaring it as a class at the top of the file. Without that, yuidoc doesn't actually detect what it should be documenting.
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/components/osf-navbar/component.js#L10

Let me know if I can help you get started with the documentation process. But for now, at very least the reusable component should tell you how to use it!